v1/plugins/logs/encoder_test.go

```diff
@@ -149,8 +149,8 @@ func TestChunkEncoderAdaptive(t *testing.T) {
 	actualScaleDownEvents := enc.metrics.Counter(encSoftLimitScaleDownCounterName).Value().(uint64)
 	actualEquiEvents := enc.metrics.Counter(encSoftLimitStableCounterName).Value().(uint64)
 
-	expectedScaleUpEvents := uint64(8)
-	expectedScaleDownEvents := uint64(3)
+	expectedScaleUpEvents := uint64(5)
+	expectedScaleDownEvents := uint64(0)
```

Comment:
Do we have a test that asserts an expected scale-dow? If not anymore, maybe we should add one.

Reply:
I raised the number of events to 1000 and now there are scale down events.